### ISSUE TITLE
How many EPOCH has been trained to reach the reported metric in the paper?

### ISSUE THREAD

**Issue by bianhao123** [swz30/MIRNetv2#19]
Thank you for your excellent work!
In the Low-light image enhancement, the reported metrics of Lol dataset are 24.74(PSNR) and 0.851(SSIM).  I want to know that how many EPOCH the model trained to reach this metrics

**Comment by bianhao123** [comment]
Can you show your train log in Lol dataset?

**Comment by Lz-2019317** [comment]
您好，你训练代码跑起来了吗？可以帮我看下吗？我训练不起来？单卡单机！
报错如下：
`ModuleNotFoundError:` No module named 'basicsr'
这个basicsr是项目文件夹，导入为沙会报错阿？

`ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 1) local_rank: 0 (pid: 28610) of binary: /home/xtzg/anaconda3/envs/pytorch1.1`
我的设置：
train.sh中设置为：
`python -m torch.distributed.run --nproc_per_node=1 --master_port=4321 basicsr/train.py -opt $CONFIG --launcher pytorch`
yaml配置文件中也修改shuffle为false
`    # data loader
    use_shuffle: false     # true
    num_worker_per_gpu: 0  # 8
    batch_size_per_gpu: 1  # 8`

**Comment by bianhao123** [comment]
> 您好，你训练代码跑起来了吗？可以帮我看下吗？我训练不起来？单卡单机！ 报错如下： `ModuleNotFoundError:` No module named 'basicsr' 这个basicsr是项目文件夹，导入为沙会报错阿？
> 
> `ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 1) local_rank: 0 (pid: 28610) of binary: /home/xtzg/anaconda3/envs/pytorch1.1` 我的设置： train.sh中设置为： `python -m torch.distributed.run --nproc_per_node=1 --master_port=4321 basicsr/train.py -opt $CONFIG --launcher pytorch` yaml配置文件中也修改shuffle为false ` # data loader use_shuffle: false # true num_worker_per_gpu: 0 # 8 batch_size_per_gpu: 1 # 8`

需要安装basicsr，见https://github.com/swz30/MIRNetv2/blob/main/INSTALL.md

**Comment by Lz-2019317** [comment]
谢谢！可以了～您对代码改进了吗？您觉得从哪方面可以进行修改？想做人脸图像的去噪，可以和给点宝贵的建议吗？不胜感激～

**Comment by yujiaolong1120** [comment]
在测试DND数据集结果时，需要将生成的结果上传官网，请问时直接上传生成的50个mat文件吗，为什么我上传以后显示错误？
